### PR TITLE
Docs: Add 7.0 topics to menu

### PR DESCRIPTION
**Pull request by oddlittlebird**
Added some 7.0 topics to the side menu.

@marefr , you might need to merge manually because of the timerange thing.

### CODE REVIEW

**Review by marcusolsson — APPROVED** [review]
LGTM

File: docs/sources/menu.yaml [hunk 1]
```diff
@@ -167,7 +169,11 @@
       name: Table
     - link: /panels/visualizations/text-panel/
       name: Text
-  - link: /panels/calculations-list/
+    - link: /panels/thresholds/
+      name: Thresholds
+   - link: /panels/inspect-panel/
```
Comment (marcusolsson) [inline docs/sources/menu.yaml:174]:
```suggestion
    - link: /panels/inspect-panel/
```

File: docs/sources/menu.yaml [hunk 2]
```diff
@@ -167,7 +169,11 @@
       name: Table
     - link: /panels/visualizations/text-panel/
       name: Text
-  - link: /panels/calculations-list/
+    - link: /panels/thresholds/
+      name: Thresholds
+    - link: /panels/inspect-panel/
+      name: Inspect panel
+      - link: /panels/calculations-list/
```
Comment (daniellee) [inline docs/sources/menu.yaml]: Once this indentation error is fixed then LGTM.

**Review by daniellee — CHANGES_REQUESTED** [review]

**Review by daniellee — APPROVED** [review]
LGTM

**Pull request merged**